sources/common/toddl.go

```diff
@@ -251,6 +255,13 @@ func CvtIndexHelper(conv *internal.Conv, tableId string, srcIndex schema.Index,
 		for _, v := range spColIds {
 			if v == k.ColId {
 				isPresent = true
+				if (conv.SpDialect == constants.DIALECT_POSTGRESQL) {
+					if(spColDef[v].T.Name == ddl.Numeric) {
+						//index on NUMERIC is not supported in PGSQL Dialect currently.
+						//Indexes which contains a NUMERIC column in it will need to be skipped.
+						return ddl.CreateIndex{}
```

Comment:
Is the purpose of this passing back an empty index ?
meaning this is a placeholder for a constant ?